foxpuppet/api/windows.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
First I agree with Dave that we should not move the files under `api/`. For Puppeteer this folder only contains Gecko-API related modules. Foxpuppet is a helper for UI interaction, so it's fine to have all in the top-level folder.

Also please do not remove files and create new ones with the same content. Instead make use of `git mv` to move the files.